### ISSUE TITLE
Ceylon plugin breaks dark theme

### ISSUE THREAD

**Issue by SyndromeDayna** [ceylon/ceylon-ide-eclipse#1816]
If dark theme is enabled Ceylon Plugin overwrites all settings located at 
`Window->Preferences->General->Editors->Text Editors->Annotations`
on every startup .

It does not change annotations colors, but changes highlighting styles.
For example it forces compilation errors to use style that fills background.

[Developer fix this plz (image 1)](http://imgur.com/a/IK7HH)
[Developer fix this plz (image 2)](http://imgur.com/a/38XZQ)

**Comment by ncorai** [comment]
Yeah, CeylonIDE has long messed up the JDT editor but in a way it's gotten worse recently. Between that and the totally muted colors I reported in #1837 I really feel like you guys should target 1.3.2 rather than 1.4

Ceylon has the grandest ambitions but keeps stumbling on the most mundane issues that make the dev lives harder and turns off potential adopters. Not everyone I know wants to install the CeylonIDE plugin for some (for the moment) secondary projects, only to have it hijack their normal environment.
